### ISSUE TITLE
Scenario spots you can get stuck on

### ISSUE THREAD

**Issue by ragafury** [WarEmu/WarBugs#9520]
in reference to this thread: 

https://www.returnofreckoning.com/forum/viewtopic.php?f=15&t=21019&start=10

TL;DR version:
There are places on scenario maps you can get punted to where you can't get out without logging out and getting quitter or getting mercy killed.
If nobody of the enemy team mercy kills you, you are stuck/trapped there until the map ends.

About the 1st Screenshot of Reikland Factory:
I am not sure if you can get out, one time I was stuck there with my zealot and unable to get out for 2-3 minutes, I was than mercy killed.

Ofc these are in the live enviroment not recreatable cause it needs some testing.

I attached the screens of the places.

If fix is not possible consider the request of the thread for a suicide command.

thank you for your hard work, have a nice weekend.
![nordenwatch - lanternpost of doom](https://cloud.githubusercontent.com/assets/26032314/25480193/0c343cea-2b48-11e7-9347-f8cdff16c255.jpg)
![not sure spot - reikland factory](https://cloud.githubusercontent.com/assets/26032314/25480195/0c3a3212-2b48-11e7-9e25-49cda9b875f4.jpg)
![steammachine of immortality - reikland factory](https://cloud.githubusercontent.com/assets/26032314/25480194/0c392304-2b48-11e7-9ec7-38c4deb015aa.jpg)

**Comment by grunbag** [comment]
![skori_267](https://cloud.githubusercontent.com/assets/24207191/25499906/58f673c8-2b8d-11e7-813e-62af0d94cb87.jpg)

inside this grill on destro bridge at talabec dam. when bomb explodes, it punt you inside the grill.

**Comment by NorKalli** [comment]
![image](https://user-images.githubusercontent.com/19868634/39390670-a7a383ec-4a4c-11e8-9a33-d3a0912fec2d.png)

**Comment by NorKalli** [comment]
![image](https://user-images.githubusercontent.com/19868634/39390682-d5fec918-4a4c-11e8-8d49-6efb65311e6b.png)

**Comment by sioding** [comment]
Fixed with Hotfix from 29.04.2017
